wasm/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+use css_inline as rust_inline;
+use wasm_bindgen::prelude::*;
+use std::borrow::Cow;
+use std::convert::TryFrom;
+use std::convert::TryInto;
+
+struct InlineErrorWrapper(rust_inline::InlineError);
+
+impl From<InlineErrorWrapper> for JsValue {
+    fn from(error: InlineErrorWrapper) -> Self {
+        JsValue::from_str(error.0.to_string().as_str())
+    }
+}
+
+struct UrlError(url::ParseError);
+
+impl From<UrlError> for JsValue {
+    fn from(error: UrlError) -> Self {
+        wasm_bindgen::throw_str(&error.0.to_string())
```

Comment:
I am not sure about this one since it doesn't return (there is `!` return type specified), but the signature implies `JsValue` return type + according to the docstring this function will not run destructors - I am not sure how WASM works here, and maybe it is not the case, but for me, it seems more reliable to use `JsValue::from_str(error.0.to_string().as_str())` instead